langlib/langlib-test/src/test/resources/test-src/arraylib_test.bal

```diff
@@ -983,10 +1000,26 @@ function testSort1() {
     "{\"id\":1,\"fname\":\"Amber\",\"fee\":10000.56,\"impact\":0.127,\"isUndergrad\":true}");
     assertFalse(studentArr == sortedArr);
 
+    assertValueEquality(sortedTup[0].toString(),
```

Comment:
Let's do the assertions in Ballerina itself. We can do it just for the additions. Also avoid asserting strings, since we lose type validations.

Reply:
Changed the assertions of string in c5daaea654d92b306868ce09652d183a764da99f